Libraries/LibWeb/WebAudio/AudioProcessingEvent.idl

```diff
@@ -0,0 +1,16 @@
+#import <DOM/Event.idl>
+#import <WebAudio/AudioBuffer.idl>
+
+[Exposed=Window]
+interface AudioProcessingEvent : Event {
```

Comment:
Could you include a spec link for this?

Relevant link: https://webaudio.github.io/web-audio-api/#AudioProcessingEvent